src/sections/Home/Partners-home/index.js

```diff
@@ -22,17 +33,69 @@ const Projects = () => {
           </SectionTitle>
         </Row>
         <Row Hcenter className="row">
-          {
-            partners.map( (partner ,index) => (
-              <Col sm={2} md={2} lg={2} key={index}>
-                <Link className="partner-card" to={partner.imageRoute}>
-                  <div className={partner.innerDivStyle}>
-                    <img src={partner.imageLink} alt={partner.name} />
-                  </div>
-                </Link>
-              </Col>
-            ))
-          }
+          <Col sm={2} md={2} lg={2}>
+            <Link className="partner-card" to="/partners#redhat">
+              <div className="partner__block__inner horizontal">
+                <StaticImage src={redhat} alt= "Redhat" objectFit="contain"/>
```

Comment:
Since all the images are `svg` in this file, let's avoid `StaticImage` and use the `img` tag itself.

Reply:
if we are not using `staticImage` then reverting this will be fine in my opinion, as it was first implemented using the `map` function which is somewhere more efficient than using individual divs, what are your thoughts on it?

Reply:
Yes, correct.

Reply:
okay will do that :)